java-squid/src/main/java/org/sonar/java/se/NotEqualRelation.java

```diff
@@ -71,6 +71,30 @@ protected Boolean impliesNotMethodEquals() {
 
   @Override
   @CheckForNull
+  protected Boolean impliesGreaterThan() {
+    return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Null is returned but a "Boolean" is expected. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2447)